docker/db.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
```

Comment:
What's the reason for removing the `version` attribute here? Can we not keep it, and just add the `name` attribute below / above?

Reply:
Sorry, just read the PR description in more depth 🤦